tidy3d/components/monitor.py

```diff
@@ -489,6 +489,12 @@ class ModeSolverMonitor(AbstractModeMonitor):
     ...     name='mode_monitor')
     """
 
+    direction: Direction = pydantic.Field(
+        "+",
+        title="Propagation direction",
+        description="Direction of waveguide mode propagation.",
```

Comment:
maybe add "along the axis defined by its normal dimension." or something to this effect.